book/src/asm/registers.md

```diff
@@ -0,0 +1,40 @@
+# Registers
+
+Registers are central to a machine. powdr supports a few types of registers:
+
+## Program counter
+
+Each machine can have at most one program counter. In the absence of a program counter, the machine is considered static, and no other register can be declared. The program counter is defined like so:
+
+```
+reg pc[@pc]
+```
+
+At each step execution step, the program counter points to the ROM line to execute.
```

Comment:
```suggestion
At each execution step, the program counter points to the ROM line to execute.
```

Reply:
I think this part may be a bit confusing. What's ROM here?

Reply:
True. Maybe function line? The distinction is that all functions are bundled into a ROM, but maybe function is good enough here?

Reply:
yea agree